host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang@upstream_nvcc_xlf.cmake

```diff
@@ -0,0 +1,119 @@
+##################################
+# !!!! This is a generated file, edit at own risk !!!!
+##################################
+
+# Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+##################################
+
+##################################
+
+# SYS_TYPE: blueos_3_ppc64le_ib_p9
+# Compiler Spec: clang@upstream_nvcc_xlf
+##################################
+
+# CMake executable path: /usr/WS1/axom/thirdparty_libs/builds/2019_06_27_22_06_03/clang-upstream_nvcc_xlf/cmake-3.9.6/bin/cmake
+
+##############
+# Compilers
+##############
+
+# C compiler used by spack
+set(CMAKE_C_COMPILER "/usr/tce/packages/clang/clang-upstream-2019.03.26/bin/clang" CACHE PATH "")
+
+# C++ compiler used by spack
+set(CMAKE_CXX_COMPILER "/usr/tce/packages/clang/clang-upstream-2019.03.26/bin/clang++" CACHE PATH "")
+
+# Fortran compiler used by spack
+set(ENABLE_FORTRAN ON CACHE BOOL "")
+
+set(CMAKE_Fortran_COMPILER "/usr/tce/packages/xl/xl-2019.06.12/bin/xlf2003" CACHE PATH "")
+
+##############
+# TPLs
+##############
+
+# Root directory for generated TPLs
+set(TPL_ROOT "/usr/WS1/axom/thirdparty_libs/builds/2019_06_27_22_06_03/clang-upstream_nvcc_xlf" CACHE PATH "")
+
+# conduit from uberenv
+set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.4.0" CACHE PATH "")
+
+# mfem from uberenv
+set(MFEM_DIR "${TPL_ROOT}/mfem-4.0" CACHE PATH "")
+
+# hdf5 from uberenv
+set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.19" CACHE PATH "")
+
+# scr not built by uberenv
+
+# raja from uberenv
+set(RAJA_DIR "${TPL_ROOT}/raja-0.7.0/share/raja/cmake" CACHE PATH "")
+
+# umpire from uberenv
+set(UMPIRE_DIR "${TPL_ROOT}/umpire-0.3.2/share/umpire/cmake" CACHE PATH "")
```

Comment:
Just a heads up -- there is a minor memory leak in umpire-0.3.2. We are probably not impacted by this since we are not currently making use of Umpire's memory pool features. But, we should probably update to the latest version of umpire soon, which also consists of some minor API changes.

Reply:
Thanks @gzagaris. Let's save that for a future PR.  Could you please add an issue with the details (e.g. which version to upgrade).